### ISSUE TITLE
ignore_http_error_codes not working

### ISSUE THREAD

**Issue by albertocastillo2001** [thp/urlwatch#341]
Hello

I am using version 2.1.5 and it seems that ignore_http_error_codes is not working, neither does ignore_connection_errors: true

I tried to set ignore_http_error_codes: 403 as currently the site shows that error message and eventually it won't. And that's when I want to get notified.

Settings are as follows. Keep in mind I am using Site-Name as an example:
```

filter: ''
kind: url
name: Site-Name
url: https://sitename/
ignore_http_error_codes: 403

```
This is the output I get all the time:
```

01. ERROR: Site-Name
===========================================================================

---------------------------------------------------------------------------
ERROR: Site-Name (https:/sitename/)
---------------------------------------------------------------------------
403 Client Error: Forbidden for url: https://sitename/
---------------------------------------------------------------------------

```
Thanks

**Comment by cfbao** [comment]
`ignore_http_error_codes` is only recently added to the master branch, and not in any released version yet.

**Comment by thp** [comment]
This feature was released in urlwatch 2.16, released on 2019-01-27.
